code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -285,7 +285,7 @@
 	center_of_mass = list("x"=16, "y"=4)
 	New()
 		..()
-		reagents.remove_reagents()
+		reagents.del_reagents()
```

Comment:
this code shouldn't still be here, update to master

Reply:
already was done but not the breach and it's fix any way